specification/sbi_atee.adoc

```diff
@@ -101,35 +101,106 @@ This operation requires the host to convert a sufficient number of non-confident
 payload for the TVM from non-confidential pages to confidential pages, and extends the corresponding measurements for the
 TVM.
 
-===== VCPU shared state enumeration
-The TSM communicates additional information about TVM exits from `sbi_tee_host_run_tvm_vcpu()` using a non-confidential
-shared memory region that's configured on a per-VCPU basis by the host. The host can also use this shared memory
-region to control and configure TVM parameters like the initial-entry point (SEPC), initial parameter, etc., and
-to respond to TVM exits.
+===== VCPU shared state
+Host needs access to some of the TVM CSRS and GPRs to handle TVM exits. For example host needs `scause` to determine
```

Comment:
`scause` isn't a good example for this since it's actually reported via CSR directly, not shared memory. Maybe `htval` instead?